README.md

```diff
@@ -112,6 +112,17 @@ Try manual installation:
         1. Enter `about:debugging` in the URL bar
         2. Click "Load Temporary Add-on" and point it at `add-on/manifest.json`
 
+You can also do the build via Docker. Run the following commands for ending up
+with a built extension inside the `build/` directory.
+
+```sh
+docker build -t ipfs-companion .
+docker run -it ipfs-companion yarn test # Make sure all tests are passing before building
+docker run -it $(pwd)/build:/usr/src/app/build ipfs-companion yarn build
```

Comment:
should this be `docker run -i -v` as per the Jenkinsfile?

Reply:
@olizilla yeah, could be. We don't really need pseudo-tty but it's handy when you want to use a bash prompt for example. I would keep it, as it's handy for debugging to just replace `yarn build` with `bash` or `sh` and you can inspect how things are inside a running container. Doesn't really break anything either.

Reply:
There's definitely a `-v` missing though

Reply:
Most certainly, thanks @lgierth